### ISSUE TITLE
SQLite RAISE does not support IGNORE

### ISSUE THREAD

**Issue by doublethink13** [sqlfluff/sqlfluff#6715]
### Search before asking

- [x] I searched the [issues](https://github.com/sqlfluff/sqlfluff/issues) and found no similar issues.

### What Happened

Running:

```bash
echo "CREATE TRIGGER block_write_after_lock
AFTER INSERT ON pod_cidr
BEGIN
SELECT RAISE(IGNORE, 'Table is locked for writes');
END;" | sqlfluff lint --dialect sqlite -
```

Gives the following error:

```bash
== [stdin] FAIL
L:   4 | P:   8 |  PRS | Line 4, Position 8: Found unparsable section:
                       | "RAISE(IGNORE, 'Table is locked for write..."
WARNING: Parsing errors found and dialect is set to 'sqlite'. Have you configured your dialect correctly?
All Finished!
```

I see that supporting `RAISE` for SQLite was implemented [here](https://github.com/sqlfluff/sqlfluff/pull/6108/files#diff-0c952c6975b7b62bd6ecbb47e4e0765d17ca8fd973c3b507df6f95e87a4882c7R318). I have arrived there from [#6101](https://github.com/sqlfluff/sqlfluff/issues/6101).

But it seems implementation is missing support for `IGNORE`? It is supported in SQLite according to [this](https://www.sqlite.org/syntax/raise-function.html) documentation.

My `sqlfuff` version is `3.3.1`. (running as `docker run -i --rm -v $PWD:/sql sqlfluff/sqlfluff:3.3.1 "$@"`)

### Expected Behaviour

No errors.

### Observed Behaviour

`IGNORE` seems to not be supported.

### How to reproduce

```bash
echo "CREATE TRIGGER block_write_after_lock
AFTER INSERT ON pod_cidr
BEGIN
SELECT RAISE(IGNORE, 'Table is locked for writes');
END;" | sqlfluff lint --dialect sqlite -
```

### Dialect

`sqlite`

### Version

`3.3.1` (running as `docker run -i --rm -v $PWD:/sql sqlfluff/sqlfluff:3.3.1 "$@"`)

### Configuration

None

### Are you willing to work on and submit a PR to address the issue?

- [x] Yes I am willing to submit a PR!

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/sqlfluff/sqlfluff/blob/main/CODE_OF_CONDUCT.md)

**Comment by WittierDinosaur** [comment]
Per the docs you linked, `IGNORE` is `sqlite` doesn't support a message afterwards. `IGNORE` on its own in brackets works

**Comment by doublethink13** [comment]
You're right, my bad, 😅 

I would improve the error message but otherwise feel free to close it
